include/deal.II/grid/manifold_lib.h

```diff
@@ -127,6 +116,131 @@ class SphericalManifold : public ChartManifold<dim, spacedim, spacedim>
 
 
 /**
+ * Manifold description for a spherical space coordinate system.
+ *
+ * You can use this Manifold object to describe any sphere, circle,
+ * hypersphere or hyperdisc in two or three dimensions, both as a co-dimension
+ * one manifold descriptor or as co-dimension zero manifold descriptor.
```

Comment:
are you saying that this can be used for "a co-dimension one manifold descriptor of a spherical surface embedded in a higher dimensional space, or as a co-dimension zero manifold descriptor for a body with positive volume"? I *think* that's what you're trying to say, but it's not entirely clear to me.